### ISSUE TITLE
BeakerNode perspectiveFactor

### ISSUE THREAD

**Issue by jonathanolson** [phetsims/fractions-intro#20]
Its usage only looks like it is using `perspectiveFactor / 2`. Can the constant factor be changed so that just `perspectiveFactor` can be used?

**Comment by veillette** [comment]
it is the width that is divided by 2 (to get the radius) and then it is multiplied by the perspectiveFactor. We should reorder the operations  to make it more explicit.

**Comment by davisv7** [comment]
@jonathanolson, we looked at the code you were concerned about and noticed that almost every use of options.beakerWidth is accompanied by a division of 2. @michaelm396 , @coledu , and I discussed this and we think it would be more intuitive to continue taking beakerWidth as an option for BeakerNode, but in the function itself to create a variable radius = beakerWidth / 2 and replace all the instances with radius. Assigning to @coledu & myself. Will assign back to @jonathanolson when committed.
